poke.io.js

```diff
@@ -295,13 +295,13 @@ function Pokeio() {
 
   // Still WIP
   self.GetFortDetails = function (fortid, fortlat, fortlong, callback) {
-    var FortMessage = new RequestEnvelop.FortDetailsRequest({
+    var FortDetailsMessage = new RequestEnvelop.FortDetailsRequest({
```

Comment:
Since you're updating this, you might as well fix the case of the first letter too to be lowercase.